pages/test_analytics/monitors.md

```diff
@@ -30,4 +30,4 @@ When you get an alert from one of your monitors, see
 
 Mark it as resolved, mute it, or leave it open.
 
-**Coming soon:** web hooks and integrations with platforms such as Slack, GitHub, and issue tracking tools so you can assign ownership of problematic tests, and see alerts in your preferred platform.
+**Added new Features:** web hooks and integrations with platforms such as Slack, GitHub, and issue tracking tools so you can assign ownership of problematic tests, and see alerts in your preferred platform.
```

Comment:
Recommend leaving this as-is, or removing the entire "coming soon" section if you're trying to avoid questions/confusion. "Added new features" to me sounds as if this is already released, which is not accurate.